website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -26,15 +26,14 @@ dbt commands are run against dbt Cloud's infrastructure and benefit from:
 The dbt Cloud CLI is available in all [deployment regions](/docs/cloud/about-cloud/regions-ip-addresses) and for both multi-tenant and single-tenant accounts (Azure single-tenant not supported at this time).
 
 - Ensure you are using dbt version 1.5 or higher. Refer to [dbt Cloud versions](/docs/dbt-versions/upgrade-core-in-cloud) to upgrade.
-- Avoid using SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections.
-- Avoid using [PrivateLink](/docs/cloud/secure/about-privatelink).
+- Note that SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections, [PrivateLink](/docs/cloud/secure/about-privatelink), and [Single sign-on (SSO)](/docs/cloud/manage-access/sso-overview) isn't supported yet.
```

Comment:
PrivateLink will be supported as of tomorrow!

Reply:
awesome! will it be compatible with the SL ? cc @aiguofer